package.json

```diff
@@ -24,6 +24,8 @@
     "react-dom": "^16.0.0"
   },
   "devDependencies": {
+    "@types/pixi.js": "^4.7.0",
+    "@types/react": "^16.0.36",
```

Comment:
Can we keep the versions consistent with `peerDepencencies`?

Reply:
Certainly.  I will address this.